### PR TITLE
add user profile and remove extra files

### PR DESCRIPTION
**Pull request by naimaseales**

### CODE REVIEW

**Comment by naimaseales** [comment]
@eliotsykes

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -15,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap-sprockets";
+@import "bootstrap";
```
Comment (eliotsykes) [inline app/assets/stylesheets/application.scss:17]: @naimaseales as discussed briefly on call today, consider taking full control of the stylesheet load order with this change:

``` diff
- *
- *= require_tree .
- *= require_self
 */
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "welcome";
+@import "users";
```

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (eliotsykes) [inline app/assets/stylesheets/users.scss:3]: `users.scss` can safely be deleted.

File: app/assets/stylesheets/welcome.scss [hunk 3]
```diff
@@ -1,3 +1,6 @@
+h1 {
+  color: red;
+}
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (eliotsykes) [inline app/assets/stylesheets/welcome.scss:6]: @naimaseales comments can be safely removed from `welcome.scss`

File: Gemfile [hunk 4]
```diff
@@ -47,3 +46,5 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+
+gem 'bootstrap-sass'
```
Comment (eliotsykes) [inline Gemfile:50]: @naimaseales Install the following gems to help with debugging efforts in future:

```
  # In development, test group:
  gem 'pry-rails' # allows you to call binding.pry to "pry open" running code
  gem 'pry-inline' # automatically shows variable values inline in rails console
  gem 'rb-readline' # rb-readline makes pry history behave on all platforms (e.g. ensures ~/.pry_history auto loads)
```

```
  # In development group only
  gem 'better_errors' # Enhanced Rails error page with built-in rails console
  gem 'binding_of_caller' # Required by better_errors
  gem 'quiet_assets' # Less noisy but easier to debug server output
```

File: app/views/users/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<div class="row">
+  <div class="col-md-4 pull-left">
+    <p>image placeholder</p> 
+  </div>
+  <div class="col-md-4 pull-right">
+    <h1>current_user.name</h1>
+    <h2>current_user.email</h2>
+  </div>
```
Comment (eliotsykes) [inline app/views/users/show.html.erb:8]: @naimaseales Use `<%= current_user.name %>` to print out the user's name

**Pull request merged**